src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -110,7 +112,11 @@ public EditLessonCommand parseEditLesson(String args, GuiState guiState) throws
             editLessonDescriptor.setDay(ParserUtil.parseDay(argMultimap.getValue(PREFIX_DAY).get()));
         }
 
-        // TODO: Need to deal with case where only start or end time is given. Currently assume both given.
+        // Prevent users from inputting only either the start or the end timings
+        if ((argMultimap.getValue(PREFIX_START).isPresent() ^ argMultimap.getValue(PREFIX_END).isPresent())) {
```

Comment:
i think no need for the double brackets here. either bracket like ((a)^(b)) or simply (a^b).

Reply:
Yup my bad, removed the extra brackets